hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -209,6 +211,7 @@ genCloseMutation (tx, _utxo) =
                 (replacePolicyIdWith Fixture.testPolicyId otherHeadId healthyOpenHeadTxOut)
                 (Just $ toScriptData healthyCloseDatum)
             ]
+    , SomeMutation Nothing MutateContestors . ChangeOutput 0 <$> mutateCloseContesters
```

Comment:
What is the purpose of this test? Just mangling the contestors might not make this fail. Asked differently.. is it failing for the right reason?

One case I can see is that we want to mutate healthy contestors by adding the party who would like to contest to it, which should trigger the check that you cannot contest twice. That's not really a property test, but on this level I don't think we can do better.

Instead, it should be simple to write a End2End test about this.

Reply:
On closeTx the list of contesters should init as empty, otherwise, an attacker could include another party to the list preventing them to contest.

This mutation test checks exactly that.

Reply:
We already have an e2e test in DirectChainSpec, is it not enought?